packages/odyssey/src/scss/base/_iconography.scss

```diff
@@ -0,0 +1,13 @@
+.ods-icon {
+  position: relative;
+  top: -0.1em;
```

Comment:
Seems like something we should try to mitigate.. any idea what causes the need to bring this up?

Reply:
The icons actually sit perfectly centered within their parent / on their line, but our text doesn't. This is a pretty common issue, sadly. Most DS's do something similar, tho usually with `px` units.

The `em` won't scale perfectly, but our icons aren't intended for much larger sizes than this.

Carbon pushed an update to their type this month that removes the need for this, so I'll be looking into their solve. We also want to resolve some other type rendering issues (check out the difference between Odyssey in Chrome and FF), but they're out of scope for this PR.